packages/tools/src/tools/displayTools/Labelmap/labelmapDisplay.ts

```diff
@@ -134,6 +134,41 @@ function removeSegmentationRepresentation(
   }
 }
 
+/**
+ * Checks if a segmentation data have the same frameOfReference as the series
+ * displayed in a given viewport
+ * @param viewport
+ * @param referencedVolumeId volume id of the segmentation reference series
+ * @returns
+ */
+function isSameFrameOfReference(viewport, referencedVolumeId) {
+  // if the referencedVolumeId is not defined, we acted as before to not break
+  // applications as referencedVolumeId is inserted in this change
+  // Can modify that in the future commits
```

Comment:
This is not true, we already had it at LabelmapSegmentationData

Reply:
Although the representationData has the field, currently the value has not been set

Reply:
Removed the comments